courses/views.py

```diff
@@ -86,9 +86,20 @@ class CourseView(DetailView):
     template_name = "course_detail.html"
 
     def get_context_data(self, **kwargs):
+        course = self.object
+        run = course.first_unexpired_run
+        product = run.products.first() if run else None
+        product_version = product.latest_version if product else None
+        enrolled = CourseRunEnrollment.objects.filter(
+            user=self.request.user, run=run
+        ).exists()
```

Comment:
Since `run` can be `None` here, should this be `if run else False`? I doubt we'll ever set `run=None`, but explicit is probably better and might as well save a query operation.